README.md

```diff
@@ -1,70 +1,99 @@
+
+_[Here](https://e-budget-plan.herokuapp.com/) live demo link_
 
-> Description the project.
 
+  Splash Screen                                 |  Transaction Page
+![screenshot](./app/assets/images/Splash.jpeg)        |   ![screenshot](./app/assets/images/Transaction.jpeg)
 
-## Built With
+  Login Page                                    |  Categories Page
+![screenshot](./app/assets/images/Log-in.jpeg)     |   ![screenshot](./app/assets/images/Categories.jpeg)
 
-- Major languages
-- Frameworks
-- Technologies used
+  Sign-Up Page                                          
+![screenshot](./app/assets/images/Sign-Up.jpeg) 
 
-## Live Demo (if available)
+# Getting Started
 
-[Live Demo Link](https://livedemo.com)
 
+_To get a local copy up and running follow these simple steps._
 
-## Getting Started
+1. Clone the repo
+   ```sh
+   git clone https://github.com/your_username_/Project-Name.git
+   ```
+2. Goto project directory
+   ```sh
+   cd recipe-app
+   ```
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+3. Configure ` change  the Env credentials with your username and password
+4. Run app
+   ```sh
+   rails server
+   ```
+   or
+   ```sh
+   rails s
+   ```
 
 
-To get a local copy up and running follow these simple example steps.
+# Testing
 
-### Prerequisites
+`bundle exec rspec` will run all the tests.
 
-### Setup
+Also, you can run `bundle exec rspec spec/` to run specific tests.
 
-### Install
+You can also run `RAILS_ENV=test rspec spec/` to run the tests in test mode.
 
-### Usage
+## Errors
 
-### Run tests
+If you encounter any errors, run the following commands.
 
-### Deployment
+- Run `rails db:drop db:create db:migrate` to drop, create and migrate a new database.
 
+- Run `rubocop && rubocop -A` to check for and fix code errors.
 
+# This project was built with
 
-## Authors
+- Ruby on Rails
 
-👤 **Author1**
+- PostgreSQL
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- Bootstrap CSS Framework
 
+# Author 
 
-## 🤝 Contributing
+👤 **Sangwa Didier**
+
+- GitHub: [Didier Sangwa](https://github.com/sangwa7)
+- Twitter: [@sangwa_7](https://twitter.com/sangwa_7)
+- LinkedIn: [Didier Sangwa](https://www.linkedin.com/in/didier-sangwa)
+
+
+# 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Donard97/recipe-app/issues).
 
-## Show your support
+# Show your support
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+# Acknowledgement
```

Comment:
- [ ] In the Acknowledgement section of the readme file, you should give appropriate credit to [Gregoire Vella](https://www.behance.net/gregoirevella) on Behance the author of the original design, as required by the Creative Commons license of the design.